packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -276,7 +282,7 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     );
   }
 
-  private _createBars(data: IChartProps, palette: IPalette): JSX.Element[] {
+  private _createBars(data: IChartProps, palette: IPalette): (JSX.Element | null)[] {
```

Comment:
when is null value possible here.

Reply:
The callback returns null instead of placeholder bar when hideValues prop is true for absolute-scale variant 
![image](https://user-images.githubusercontent.com/110246001/212726202-23a288ff-8054-46b7-a070-0fbe9d5a04f5.png)

Reply:
But hidevalue hides just the labels not the entire bar. Then why is create bars returning null from top level

Reply:
Got the point. Fixed it by replacing null with empty SVG text element